src/utils/intuitAPI.ts

```diff
@@ -337,6 +338,33 @@ export default class IntuitAPI {
     return item
   }
 
+  async _createPayment(payload: QBPaymentCreatePayloadType) {
+    console.log('IntuitAPI#createPayment | payment creation start')
+    const url = `${intuitBaseUrl}/v3/company/${this.tokens.intuitRealmId}/payment?minorversion=${intuitApiMinorVersion}`
+    const payment = await this.postFetchWithHeaders(url, payload)
+
+    if (!payment)
+      throw new APIError(
+        httpStatus.BAD_REQUEST,
+        'IntuitAPI#createPayment | message = no response',
+      )
+
+    if (payment?.Fault) {
+      console.error({ Error: payment.Fault?.Error })
+      throw new APIError(
+        httpStatus.BAD_REQUEST,
+        'IntuitAPI#createPayment | Error while creating payment',
+        payment.Fault?.Error,
+      )
+    }
+
+    console.log(
```

Comment:
```suggestion
    console.info(
```
Using console.info for informational logs, and console.log for debugging reduces the chances of commiting debug logs and logging unintended data in production 🙏